src/helpers/useMediaBrowserMetaData.ts

```diff
@@ -0,0 +1,46 @@
+import api from "@/plugins/api";
+import {
+  MediaItem,
+  PlayerMedia,
+  PlayerState,
+  QueueItem,
+  Track,
+} from "@/plugins/api/interfaces";
+import { store } from "@/plugins/store";
+import { onMounted, watch } from "vue";
+
+function playerMediaToMetadata(item: QueueItem) {
+  let artist: string | undefined;
+  let album: string | undefined;
+
+  //here we cast to Track to access properties that are only available on Track
+  if (item.media_item!.media_type === "track") {
+    const currentMedia = item.media_item as Track;
+    artist = currentMedia.artists.map((a) => a.name).join(", ");
```

Comment:
What about the case where a non-track item is being played, such as a radio station or some raw audio file ?
I think the code should handle that instead of crashing

Reply:
i am not deep enough in the codebase. as far as i understand this code shouldnt crash in those cases and rather just show the file name or have i got something wrong there?

Reply:
Yeah, I just spotted you already check if the media item is track, I missed that part last time. So all good.